packages/react-chat/src/contexts/RuntimeContext/useRuntimeState.ts

```diff
@@ -149,6 +153,7 @@ export const useRuntimeState = ({ assistant, config }: Settings) => {
       addTurn,
       feedback: runtime.saveFeedback,
       setStatus,
+      setAutostart,
```

Comment:
might not need to expose this if we move all the logic into `launch`

Reply:
made a ticket for this https://linear.app/voiceflow/issue/DX-1025/refactor-chat-modes-into-stitches-themes